common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -106,8 +106,8 @@ object Messages {
     /** Error for meta api. */
     val propertyNotFound = "Response does not include requested property."
     def invalidMedia(m: MediaType) = s"Response is not valid '${m.value}'."
-    val contentTypeNotSupported = "Content type not supported."
-    val contentTypeRequired = "Content type is missing."
+    val contentTypeNotSupported = """Content type must be specified and one of ["json", "html", "http", "text"]."""
```

Comment:
This is a packaging issue - the messages are in the common project but the set is in the controller project. As you suggested, we should split these messages and localize them to their respective services (controller vs invoker). Leaving as is until the messages are refactored.